netlify.toml

```diff
@@ -81,6 +81,11 @@
   to = "/docs/details/allowlisting-roadie-traffic/"
   force = true
 
+[[redirects]]
+  from = "/docs/getting-started/openapi-specs"
+  to = "/docs/api/authorization/"
```

Comment:
Actually, this doesn't make any sense. 

The broken page is about how to add API specs to the catalog. The page you're replacing it with is about authenticating with the Roadie APIs.

These are completely unrelated. You need to find a different page to redirect to.

Reply:
Will I go back to what i had?